source/pgg_mode/PGGHost.h

```diff
@@ -111,7 +113,11 @@ class PGGHost: public Host {
     Host::DistribResources(resources);
 
     for(size_t i=0; i < syms.size(); i++){
-      double hostPool = syms[i]->ProcessPool();
+      emp::Ptr<PGGSymbiont> sym = syms[i].DynamicCast<PGGSymbiont>();
+      if (sym == nullptr) {
+        throw "Non-PGG symbiont in PGG host";
+      }
+      double hostPool = sym->ProcessPool();
```

Comment:
This is the worst part of this change in my opinion, unfortunately I haven't really found any way to get around it without either adding stubs to BaseSymbiont (so the "Organism has every method needed by any mode" approach used before this PR) or using far too many templates.